posts/inside-rust/2023-09-19-crates-io-usage-policy-rfc.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: crates.io Policy Update RFC
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This has caused a lot of controversial discussion and most of the suggested improvements have since been integrated into the proposal.
+
+At the last team meeting the crates.io team decided to move the RFC forward and start the **final comment period** process.
+
+We have been made aware by a couple of community members though that the RFC might not have been visible enough in the Rust community. We hope that this blog post changes that.
+
+We invite you all to review the RFC and let us know if there are still any major concerns with these proposed policies.
+
+Here is a quick **TL;DR**:
+
+- The current policies are quite vague on a couple of topics. The new policies are more explicit.
+- Name squatting is allowed, but only to a certain degree and if you have a good reason for it.
```

Comment:
Nitpicking the wording a bit here, admittedly, but I feel like this captures the intent a little better:

```suggestion
- Reserving names is allowed, but only for a limited period of time while genuine development occurs.
```

Reply:
Sure, let's go with that. :+1:

(I'll admit `rg` wasn't really on my mind here — I think it's a legit case, but I also think it's such a rare mixture of factors that I don't particularly want to hone in on that case too much when we're talking generally.)